secrets/values.yaml

```diff
@@ -130,6 +133,9 @@ idds:
 
 bigmon:
   enabled: true
+  # If this is set to true, certificates will not be ingested from certs on-disk,
+  # See the existingTlsSecret value in bigmon and panda charts where the secret name can be set.
+  existingTlsSecret: true
```

Comment:
Hi @pllopis - do you mind if we do not set this true by default? We do not use a cert manager in our k8s based deployment either at CERN nor at SLAC so far - I am afraid that a third party might try to deploy this and the certs places under files/ will be ignored..

Reply:
Indeed! Sorry that's a mistake, I meant to have the default be false to keep the backwards compatibility with everything the same way you had until now, but I forgot to change it back after testing.

Do you want me to open a new PR?